recipes/pyessv/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pyessv" %}
+{% set version = "0.9.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 043949af39a00c0c01ba69ad27709b9ca03086166bd5aa4e279264b9ed495b57
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pytest
+    - tox
```

```suggestion
    - tox
    - __unix
```
This would be the way to restrict it to unix.

---

Thank you! But no need here, it's OS independent.